stripe/src/main/java/com/stripe/android/PaymentSessionConfig.kt

```diff
@@ -195,6 +198,14 @@ data class PaymentSessionConfig internal constructor(
             this.shouldShowGooglePay = shouldShowGooglePay
         }
 
+        /**
+         * @param canDeletePaymentOptions if `false`, user could not remove payment method
+         * by swiping right on the selected payment method in [PaymentMethodsActivity]
+         */
+        fun setCanDeletePaymentOptions(canDeletePaymentOptions: Boolean): Builder = apply {
+            this.canDeletePaymentOptions = canDeletePaymentOptions
+        }
```

Comment:
I think we can rename this to `canDeletePaymentMethods` because it's more consistent with our naming conventions.

Reply:
done